### PR TITLE
Added pry-byebug instructions to pry debugging

### PR DESCRIPTION
**Pull request by berubenic**
Updated debugging with pry to include pry-byebug. Added an example to go along.

### CODE REVIEW

File: ruby_programming/basic_ruby/debugging.md [hunk 1]
```diff
@@ -103,10 +103,12 @@ p "Using p:"
 p []
 ~~~
 
-### Debugging with Pry
+### Debugging with Pry and Pry-byebug
```
Comment (rlmoser99) [inline ruby_programming/basic_ruby/debugging.md:106]: I meant for this lesson to replace `Pry` with `Pry-byebug`. I apologize for this miscommunication, I re-read my earlier comment and understand how you come to a different conclusion. `Pry` is a dependent of `Pry-byebug`, so there is no need to install both. In addition, all the references to `Pry` will need to be changed to `Pry-byebug`.

File: ruby_programming/basic_ruby/debugging.md [hunk 2]
```diff
@@ -149,6 +151,40 @@ yell_greeting("bob")
...
+     5: def yell_greeting(string)
+     6:   name = string
+     7: 
+     8:   binding.pry
+     9: 
+ => 10:   name = name.upcase
+    11:   greeting = "WASSAP, #{name}!"
+    12:   puts greeting
+    13: end
+
```
Comment (rlmoser99) [inline ruby_programming/basic_ruby/debugging.md:166]: In this code example, I would not duplicate the method. Instead start at line 11 of the example `[1] pry(main)> name`. Add a line for the value of `greeting` (from the previous instructions). Then continue with your instructions on `next`.

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
Thanks for taking the time to expand on this debugging lesson. If you make changes on a branch of your fork, it will be easier for you to update files, without having to submit a new PR.

File: ruby_programming/basic_ruby/debugging.md (hunk 1 above)
Comment (berubenic) [reply ruby_programming/basic_ruby/debugging.md]: I left the first sentence as is, as I think it is worth mentioning pry. Changed other instances of `Pry` with `Pry-byebug` though it does feel a bit redundant towards the end.

**Comment by berubenic** [comment]
> Thanks for taking the time to expand on this debugging lesson. If you make changes on a branch of your fork, it will be easier for you to update files, without having to submit a ne

**Review by rlmoser99 — APPROVED** [review]

**Comment by rlmoser99** [comment]
Thanks for taking the time to update the debugging lesson with pry-byebug. I did make a few small changes to the wording that I felt was important to mention.

**Pull request merged**